src/SplitScreen/utils.js

```diff
@@ -0,0 +1,170 @@
+import { fromCircle } from 'ol/geom/Polygon'
+import * as olProj from 'ol/proj'
+
+import turfDistance from '@turf/distance'
+import turfBearing from '@turf/bearing'
+import turfDestination from '@turf/destination'
+
+const defaultDataProjection = 'EPSG:4326'
+
+export default function turf (turfFunc, argArray, projection = 'EPSG:3857') {
+  const transformedArgs = argArray.map((arg) => {
+    return transform(arg, projection, true)
+  })
+  const turfResults = turfFunc.apply(this, transformedArgs)
+
+  return transform(turfResults, projection, false)
+}
+
+export function coordValidator (coord) {
+  return coord.map((val) => {
+    return isNaN(val) ? 0 : val || 0
+  })
+}
+
+export function getCoordinates (geometry, optCircle = false) {
+  switch (geometry.getType()) {
+    case 'GeometryCollection':
+      return geometry.getGeometries().map(geom => getCoordinates(geom, optCircle))
+    case 'Circle':
+      return optCircle ? geometry.getCenter() : fromCircle(geometry).getCoordinates()
+    default:
+      try {
+        return geometry.getCoordinates()
+      } catch (e) {
+        return undefined
+      }
+  }
+}
+
+export const getHeading = (coordinate, index, allowNegative = true) => {
+  const absCoord = Math.abs(coordinate)
+  const coord = allowNegative ? coordinate : absCoord
+
+  if (index === 0 && absCoord <= 180) {
+    if (coordinate > 0) {
+      return `${coord}˚ E`
+    } else {
+      return `${coord}˚ W`
+    }
+  } else if (coordinate > 0 && absCoord) {
+    return `${coord}˚ N`
+  } else {
+    return `${coord}˚ S`
+  }
+}
+
+export function coordDiff (coord1, coord2, view) {
```

Comment:
This might be better refactored to take the `projection` directly instead of a `view` that we only use to call `getProjection` from... more general-purpose that way and have less property-envy